protos.h

```diff
@@ -192,8 +192,8 @@ void mutt_message_to_7bit(struct Body *a, FILE *fp);
 void mutt_mktemp_full(char *s, size_t slen, const char *prefix, const char *suffix, const char *src, int line);
 #define mutt_mktemp_pfx_sfx(a, b, c, d) mutt_mktemp_full(a, b, c, d, __FILE__, __LINE__)
 #define mutt_mktemp(a, b) mutt_mktemp_pfx_sfx(a, b, "neomutt", NULL)
-FILE *mutt_mkstemp_full(char *s, size_t slen, const char *src, int line);
-#define mutt_mkstemp(a, b) mutt_mkstemp_full(a, b, __FILE__, __LINE__)
+FILE *mutt_mkstemp_full(const char *src, int line);
+#define mutt_mkstemp() mutt_mkstemp_full(__FILE__, __LINE__)
```

Comment:
Change this to:
`#define mutt_mkstemp() mutt_mkstemp_full(__func__, __FILE__, __LINE__)`